include/rocksdb/utilities/object_registry.h

```diff
@@ -116,11 +116,31 @@ class ObjectLibrary {
   std::string id_;
 };
 
+#endif  // ROCKSDB_LITE
 // The ObjectRegistry is used to register objects that can be created by a
 // name/pattern at run-time where the specific implementation of the object may
 // not be known in advance.
 class ObjectRegistry {
+ private:
```

Comment:
I may prefer put the changes of object_registry.h in another PR. It is not related to the changes in customziable_utl.h right?